### DISCUSSION TITLE
Kalo ada mau nanya, tanya sini

### DISCUSSION THREAD

**Discussion by irsyaad06** [KOSATAN7/embrioclass#10] [Q&A]
ask the question that you cant answer

**Comment by AlfariziKeane** [comment]
ini mulainya kapan?

**Reply by robertarsyah** [reply]
sekarang

**Reply by Rifqi1012** [reply]
Bolehh langsung dimulai

**Reply by AlfariziKeane** [reply]
baik kang

**Reply by AlfariziKeane** [reply]
di index.html yang udah ada kan?

**Reply by NaufalRamdhanRizqika** [reply]
belum cek

**Reply by irsyaad06** [reply]
@AlfariziKeane iyaa hilmy betul sekali

**Comment by irsyaad06** [comment]


**Reply by irsyaad06** [reply]
ss

**Comment by robertarsyah** [comment]
ini full di style?

**Reply by Rifqi1012** [reply]
Iya pake tailwind yaa bukan css native

**Comment by AlfariziKeane** [comment]
cara biar listnya namanya ada di samping gambar gimana?

**Reply by iqbaljnr** [reply]
bingung

**Reply by AlfariziKeane** [reply]
blog ahh ampun

**Comment by iqbaljnr** [comment]
itu gimana cara memperbesar gambar task 1 anying

**Reply by robertarsyah** [reply]
di ubah ukuran nya

**Comment by robertarsyah** [comment]
info dong itu yang kosatan warna apa indigo,blue apa sky?

**Reply by Rifqi1012** [reply]
Indigo

**Comment by NaufalRamdhanRizqika** [comment]
![image](https://github.com/KOSATAN7/embrioclass/assets/78456483/addb370b-99d4-4608-bdd3-87dd0859c8af
ini kenapa
